plugins/destination/postgresql/client/migrate.go

```diff
@@ -246,11 +261,6 @@ func (c *Client) createTableIfNotExist(ctx context.Context, table *schema.Table)
 		sb.WriteString("_cqpk PRIMARY KEY (")
 		sb.WriteString(strings.Join(primaryKeys, ","))
 		sb.WriteString(")")
-	} else {
```

Comment:
This was part of the append mode workflow, as we use `_cq_id` instead of the real PK here.
Is it removed now?

Reply:
this still stands

Reply:
resolved in https://github.com/cloudquery/cloudquery/pull/7819#issuecomment-1428288001